server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessageListMethodStepdefs.java

```diff
@@ -59,6 +61,26 @@ private GetMessageListMethodStepdefs(MainStepdefs mainStepdefs, UserStepdefs use
         this.messagesMethodStepdefs = messagesMethodStepdefs;
     }
 
+    @When("^the user asks for message list in mailboxes \"([^\"]*)\" with flag \"([^\"]*)\"$")
+    public void getMessageList(List<String> mailboxes, String flag) throws Exception {
```

Comment:
It could be great to add a "stateless" version where you can specify the user

Reply:
I changed